### PR TITLE
Dokka-based documentation

### PR DESCRIPTION
**Pull request by sschr15**

### CODE REVIEW

**Comment by sschr15** [comment]
[Initial renders](https://github.com/QuiltMC/quilt-kotlin-libraries/files/8954099/initial-renders.tar.gz)

The `htmlCollector` directory is a version where every module is put together into one.

The `htmlMultiModule` directory is a version where each module has its own menu item

**Comment by sschr15** [comment]
I'm now hosting copies of the docs for both [`htmlMultiModule`](https://qkl.docs.concern.i.ng/htmlMultiModule/) and [`htmlCollector`](https://qkl.docs.concern.i.ng/htmlCollector/) and would like people to vote on which to include in documentation jars (or if I should make both into doc jars?)

**Comment by sschr15** [comment]
I now offer HTML Multi Module and Javadoc-style versions with a simple [index page](https://qkl.docs.concern.i.ng/). HTML Collector has gone away, and the Gradle project now supports jarfile creation.

**Comment by sschr15** [comment]
Issue with side bar on small screens, will fix when I get home

**Review by SilverAndro — APPROVED** [review]

File: codeformat/dokka/styles/prism.css [hunk 1]
```diff
@@ -0,0 +1,585 @@
...
+	--green-40:      #50FA7B66;
+	--orange-40:     #FFB86C66;
+	--pink-40:       #FF79C666;
+	--purple-40:     #BD93F966;
+	--red-40:        #FF555566;
+	--yellow-40:     #F1FA8C66;
+}
+
+pre::-webkit-scrollbar
+{
```
Comment (NoComment1105) [inline codeformat/dokka/styles/prism.css:70]: Is this bracket placement the standard for css? Else I think it should mirror what we do in the rest of the codebase

**Review by NoComment1105 — COMMENTED** [review]
Just one this, otherwise looks great

File: codeformat/dokka/styles/prism.css (hunk 1 above)
Comment (sschr15) [reply codeformat/dokka/styles/prism.css:70]: The Prism theme is taken from [dracula](https://draculatheme.com/prism) with the file itself being taken from [unpkg](https://unpkg.com/dracula-prism@2.1.13/dist/css/dracula-prism.css) in order to work with Dokka. I left it unmodified, but due to how Dokka is set up, I can't simply apply the stylesheet link the way it's intended.

Comment (NoComment1105) [reply codeformat/dokka/styles/prism.css:70]: o7 all good

**Review by NoComment1105 — APPROVED** [review]

**Review by PotatoPresident — APPROVED** [review]
This looks super cool

**Review by Oliver-makes-code — APPROVED** [review]

**Pull request merged**